lib/snapshot/snapshot_fuchsia.dart

```diff
@@ -14,3 +16,22 @@ import 'dart:math';
 import 'dart:mozart.internal';
 import 'dart:typed_data';
 import 'dart:ui';
+
+// These libraries are optional. They are included in the core snapshot and
```

Comment:
Maybe not in this CL, but we should consider adding a README.md in this directory explaining how to create/consume a snapshot for a different app profile.